### ISSUE TITLE
Improve error from Toolbox\Blueprints\BlueprintForm->doLoad()

### ISSUE THREAD

**Issue by OleVik** [getgrav/grav#2782]
With the newer, explicit [erroring out in Toolbox](https://github.com/rockettheme/toolbox/blame/develop/Blueprints/src/BlueprintForm.php#L496-L498), which was implemented from static analysis, the error - `\InvalidArgumentException(__METHOD__ . '(): Parameter #1 does not contain array of filenames');` - is rather unspecific.

It would be beneficial to include the `$filename` to tell the developer/user what caused this error specifically. As [shown in a recent error-report](https://discourse.getgrav.org/t/parameter-1-does-not-contain-array-of-filenames/10978), this can be confusing and obscure.

Tagging @mahagr [by request](https://discordapp.com/channels/501836936584101899/506916956637495306/662705500093153281). Feel free to move to Toolbox-repo if desirable, just thought it more pertinent and potentially time-sensitive for Core if it is a pervasive error.

**Comment by SteveBaumann** [comment]
I receive this error when clicking "Add" from the Plugins page in Dashboard. The Plugins page itself renders properly, listing all the installed plugins, but the page that should show all available plugins delivers the Crikey! page, stating there are no filenames in the array. 
![image](https://user-images.githubusercontent.com/11002590/71792763-cb6f1c80-2fee-11ea-820b-545d5848d885.png)

**Comment by SteveBaumann** [comment]
To replicate the bug, and with the help of Ole & Ricardo, I tried a test, using both PHP 7.2.19 and 7.3.7 (the 2 versions available on my account at my web hosting provider.

I created a new directory; downloaded a copy of the 1.7 rc3 package that's available from the getgrav.org website, and FTP'd it to the new directory. 

I created an admin account, went to the Plugins page, and upon clicking the Add button, get the Crikey page (as above), even with that bare setup, with nothing extra added at all. 

After deleting everything from that directory to empty, I FTP'd a copy of the 1.7 repo, current as of this morning, into that directory, and used Terminal to install the dependencies, 2 user accounts, and the Admin Plugin. I logged in, clicked into the Plugins page, and clicked the Add button, and the list of available plugins displays normally. 

So it stands to reason, that there is something in the full initial rc3 package, that does not exist in the repo, that is causing a conflict, at least with PHP 7.2.19 and 7.3.7 - my hosting provider has not yet made PHP 7.4 available to use, for me to be able to test it.

**Comment by rhukster** [comment]
@SteveBaumann did you not see the message above the @mahagr fixed this 2 weeks ago in toolbox? You can just update the composer.lock file from the commit and do a composer update to get the fixed version of toolbox.

**Comment by SteveBaumann** [comment]
Hi Andy,  I already did that, and it didn't fix anything. Ole and I have been going over it. I actually tried overwriting composer.lock file with the current version from the repo, with no improvement. Even grabbed the entire current 1.7 repo and uploaded it over the base rc3 setup and still get the same error.
Only with a bare install of the repo from Command Line does it work for me without errors.
